src/Bowling.js

```diff
@@ -0,0 +1,43 @@
+function Bowling () {
+  this.rolls = []
+}
+Bowling.prototype.roll = function (pins) {
+  this.rolls.push(pins)
+}
+Bowling.prototype.score = function () {
+  var result = 0
+  var rollIndex = 0
+  var bowling = this
```

Comment:
You declare this variable in the function but don't use it. Maybe move it into the constructor if you need it elsewhere? Is it necessary to have a var = this?